apps/blazor-test-app/Blazor-Test-App.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="Microsoft.Graph" Version="5.6.0" />
     <PackageReference Include="Microsoft.Fast.Components.FluentUI" Version="1.5.3" />
     <PackageReference Include="Microsoft.TeamsFx" Version="2.2.*" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.21.0" />
```

Comment:
the description of the PR says that a deprecated version of this dependency was removed, but it seems like a dependency was added, was the old version removed in a different PR